x/concentrated-liquidity/tick.go

```diff
@@ -49,6 +63,24 @@ func (k Keeper) crossTick(ctx sdk.Context, poolId uint64, tickIndex int64) (liqu
 		return sdk.Dec{}, err
 	}
 
+	pool, err := k.getPoolById(ctx, poolId)
+	if err != nil {
+		return sdk.Dec{}, err
+	}
+
+	newSecondsInactive := time.Duration(ctx.BlockTime().Sub(pool.GetTimeOfCreation())) + tickInfo.SecondsInactive
+	tickInfo.SecondsInactive = newSecondsInactive
+
+	// Update seconds per liquidity outside
+	// fmt.Printf("Seconds inactive: %v \n", tickInfo.SecondsInactive.Seconds())
+	// fmt.Printf("Liquidity gross: %v \n", tickInfo.LiquidityGross)
+	tickInfo.SecondsPerLiquidityOutside = sdk.MustNewDecFromStr(fmt.Sprintf("%f", tickInfo.SecondsInactive.Seconds())).Quo(tickInfo.LiquidityGross)
```

Comment:
Just to clarify, is `SecondsInactive` "seconds outside" and `LiquidityGross` liquidity inside the current tick?

Reply:
Correct